tests/common/load_generator.hpp

```diff
@@ -20,6 +22,10 @@ inline double_t constFunction(double_t x) {
   return 10;
 }
 
+inline double_t const1Function(double_t x) {
```

Comment:
Some compilers will bark at you for the unused argument. And - can't we inline this with a lambda?